plugins/Live/templates/getLastVisitsStart.twig

```diff
@@ -85,7 +85,7 @@
 <span class="tooltip-action-server-time">{{ action.serverTimePretty }}</span>
     {% if action.timeSpentPretty is defined %}<span class='tooltip-time-on-page'>{{ 'General_TimeOnPage'|translate }}: {{ action.timeSpentPretty|raw }}</span>{% endif %}
 {%- endset %}
-                                    <img class='iconPadding' src="{{ action.iconSVG|default(action.icon) }}" title="{{- title|e('html_attr') -}}"/>
+                                    <img class='iconPadding' src="{{ action.iconSVG|default(action.icon) }}" title="{{- title -}}"/>
```

Comment:
this one might actually contain html and thus the html becomes invalid, causing vue to fail loading anything. Is there a reason why you removed the `e('html_attr')` in various cases? I've seen that on some other PRs as well, but in most cases, that was in combination with `|json_encode`, which makes the escaping unneeded...

Reply:
@sgiehl My understanding was that by default `|e('html')` is applied, which would encode eg, `<` as `&lt;`, etc. and that would be enough here. But I guess autoescape isn't enabled for twig? I'm not sure how to explain that. I added `|e('html')` and it seems to work. Let me know if I'm wrong.

Reply:
Not sure regarding autoescaping. Can't remember we turned that off. But `|e('html)` isn't the same as `|e('html_attr')` anyway. When using variables in html attributes we should always prefer `|e('html_attr')`, as this will only escape those chars, that might break the attribute value, while the html escaper also escapes `<`, `>` and some other chars.

Reply:
Ok, figured out why autoescape isn't being invoked. It appears to only be used when outputting a template variable. But title is set using `{% set %}` with some raw html, and that html does not get escaped when title is used. I don't know why twig is designed like that, but that is why an explicit filter is required here and not elsewhere.

I will leave the html/html_attr discussion to the other PR.